mod_reforged/config/items.nut

```diff
@@ -0,0 +1,27 @@
+::Reforged.Items <- {
+	function isDuelistValid( _weapon )
+	{
+		if (_weapon.isEquipped())
+			return _weapon.getContainer().getActor().getSkills().getAttackOfOpportunity().isDuelistValid();
+
+		local copy = ::new(::IO.scriptFilenameByHash(_weapon.ClassNameHash));
+		local player = ::MSU.getDummyPlayer();
+		player.equip(copy);
+		local ret = player.getSkills().getAttackOfOpportunity().isDuelistValid();
+		player.unequip(copy);
+		return ret;
+	}
+
+	function getSkills( _item )
```

Comment:
I don't see this function being used in this branch.
Have you planned anything for this?

Reply:
Yes it is not being used. Just seemed like a useful function to have in case any of our team members or submodders want to get a list of the default skills of an item.